compiler/rustc_target/src/spec/aarch64_apple_darwin.rs

```diff
@@ -1,5 +1,7 @@
-use super::apple_base::{macos_llvm_target, opts, Arch};
-use crate::spec::{FramePointer, SanitizerSet, Target, TargetOptions};
+use crate::spec::{
+    base::apple::{macos_llvm_target, opts, Arch},
+    FramePointer, SanitizerSet, Target, TargetOptions,
+};
```

Comment:
Elsewhere in rustc we don't use nested `use` afaik.

Reply:
I think we might in some places, though admittedly it is less common, but unless you've got a one-liner that I can change these all with, they can stay this way until we've got a formatter or something like that fixing it and enforcing a standard - I don't want to go through all those files again just for a minor formatting preference.

Reply:
Rustfmt has the unstable import_granularity config: https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=import#imports_granularity Using the `module` value would work I think.

Try `build/host/rustfmt/bin/rustfmt --config imports_granularity=module compiler/rustc_target/src/spec/**/*.rs`.

Reply:
I would also prefer to keep imports as is, and do the `super::apple_base` -> `super::base::apple` transformation instead.
The PR is already too large without the import changes.

Reply:
I used nested a lot so in the parts of rustc I work on, they occur regularly.